web/src/main/java/org/akaza/openclinica/controller/ImportController.java

```diff
@@ -171,11 +171,9 @@ public ImportController() {
             return new ResponseEntity(ErrorConstants.ERR_STUDY_NOT_EXIST, HttpStatus.OK);
```

Comment:
I know this was preexisting but show we start refactoring whenever possible since "description" is deprecated with this?
@Api(tags = {"Swagger Resource"})
@SwaggerDefinition(tags = {
    @Tag(name = "Swagger Resource", description = "Write description here")
})

Reply:
I think we should switch all of the APIs at once instead of having a mix of both.